### ISSUE TITLE
Support annotations for ping services in the Helm chart

### ISSUE THREAD

**Issue by TBBle** [googleforgames/agones#1491]
**Is your feature request related to a problem? Please describe.**

Running up on AWS, we want to put an annotation on the `LoadBalancer` services to use NLB instead of ELB. This is _necessary_ for udp (#1148) although not sufficient.

This also lets us apply annotations for external-dns to get nice auto-registered DNS entries for the ping services, from the Helm config instead of later.

**Describe the solution you'd like**

Being able to do this in values.yaml

```yaml
agones:
  ping:
    udp:
      annotations:
        external-dns.alpha.kubernetes.io/hostname: agones-udp-ping.example.com
        service.beta.kubernetes.io/aws-load-balancer-type: nlb
    http:
      annotations:
        external-dns.alpha.kubernetes.io/hostname: agones-http-ping.example.com
        service.beta.kubernetes.io/aws-load-balancer-type: nlb
```

This should be a reasonably simple change in the Helm chart, and is pretty common in other charts. We just need to copy the blob of YAML in the annotations value into the Service.

**Describe alternatives you've considered**

The workaround, after installation, is to use `kubectl annotate`. Although it seems from #1148 that this won't work for `service.beta.kubernetes.io/aws-load-balancer-type`, and I haven't yet tested it with `external-dns.alpha.kubernetes.io/hostname`

**Additional context**

I haven't looked, but it's probably generally-useful to be able to do this for Services in a Helm chart. Allocator would definitely benefit too (also being a `LoadBalancer` by default), although I have turned that off in my deployment here.

**Comment by aLekSer** [comment]
Hello, nice to hear that there is a workaround for UDP, as this was the main issue left on EKS.

**Comment by aLekSer** [comment]
There is an issue to track exact that annotation:
https://github.com/kubernetes/kubernetes/pull/80394

**Comment by TBBle** [comment]
Sorry, I probably shouldn't have included the NLB annotation in the udp-ping block, since that's still blocked by #1148 (and upstream https://github.com/kubernetes/kubernetes/issues/79523).

Although NLB is nice-to-have, the more-important use-case for me is `service.beta.kubernetes.io/aws-load-balancer-additional-resource-tags` as we use tags to assign costs to projects, and my Agones load-balancer currently falls into the "Work it out by hand" bucket.

**Comment by aLekSer** [comment]
I have created initial bool version to support initial annotations here:
https://github.com/aLekSer/agones/tree/helm-service-annotations
But I will rewrite it to support general annotations (not only two listed in the description) for Ping services.
